pkg/gateway/api/api.go

```diff
@@ -1239,3 +1242,14 @@ func (s *gatewayService) containsInvalidTimestampError(errs map[string]*register
 	}
 	return false
 }
+
+func (s *gatewayService) filterOutArchivedFeatures(fs []*featureproto.Feature) []*featureproto.Feature {
+	result := make([]*featureproto.Feature, 0)
+	for _, f := range fs {
+		if f.Archived {
+			continue
+		}
+		result = append(result, f)
+	}
+	return result
+}
```

Comment:
This function was temporarily added. I will remove it when I implement a differential evaluation feature.